lib/service-worker.js

```diff
@@ -36,6 +36,10 @@ var ServiceWorker = (function () {
       throw new Error('OfflinePlugin: ServiceWorker.output option must be a relative path, ' + 'but an absolute path was passed');
     }
 
+    if (typeof options.navigateFallbackURL === 'undefined' && typeof options.navigateFallbackForRedirects !== 'undefined') {
```

Comment:
`options.navigateFallbackForRedirects` won't really be `undefined` until user sets it to `undefined` since in default options it's `true` (no need to fix this, just pointing to it. see review message for details)